llvm/lib/Target/AArch64/AArch64AsmPrinter.cpp

```diff
@@ -2085,6 +2099,37 @@ void AArch64AsmPrinter::emitMachOIFuncStubHelperBody(Module &M,
                                    .addImm(2),
                                *STI);
 
+  if (TM.getTargetTriple().isArm64e()) {
+    //   autibsp
```

Comment:
Leave the tail-call checking for the general tail-call checking patches, no?

Reply:
Is there a PR up for that that I should tack this bit onto the end of?

Reply:
There is not, but send me your patch and I'll tack it onto the relevant commit (and split the tail-call check emission into a helper)

Reply:
https://github.com/llvm/llvm-project/pull/84704/commits/ec43ccba4484e550f2e8732be8fc00f3110772ba